apps/gp2-server/src/routes/public/user.route.ts

```diff
@@ -39,24 +39,28 @@ export const userRouteFactory = (userController: UserController): Router => {
 const mapUserToPublicUser = (
   user: gp2Model.UserResponse,
 ): gp2Model.PublicUserResponse => ({
-  id: user.id,
-  displayName: user.displayName,
+  avatarUrl: user.avatarUrl,
+  biography: user.biography,
+  city: user.city,
   country: user.country,
   degrees: user.degrees,
+  displayName: user.displayName,
   firstName: user.firstName,
+  id: user.id,
   lastName: user.lastName,
-  publishDate: user.createdDate,
-  avatarUrl: user.avatarUrl,
-  biography: user.biography,
-  city: user.city,
+  lastModifiedDate: user.lastModifiedDate,
   middleName: user.middleName,
+  institution: user.positions[0]?.institution,
+  title: user.positions[0]?.role,
```

Comment:
And title is confusing because in one part of the ticket says it should be the role in CMS and in other it says it should be Degree, but we're not returning either 😅 
 
![Screenshot 2024-06-04 at 07 38 27](https://github.com/yldio/asap-hub/assets/16595804/4011abea-a61e-4606-8a1c-e05771e6af1f)


![Screenshot 2024-06-04 at 07 38 33](https://github.com/yldio/asap-hub/assets/16595804/98304b17-a8c0-4d02-8dd0-e784625fa74c)

Reply:
yeah this is strange cause degrees are actually already present. I added a title and referenced a role there, I think I am just going to leave it and if they decide it was the degree they were after then they can just do that.